scripts/schmidt_comparison.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(0,'../chamber/models')
+
+from CoolProp import HumidAirProp as hap
+import pandas as pd
+
+import props
+
+
+results = {'Pressure (Pa)': [], 'Temperature (K)': [],
+					'Ralative Humidity': [], 'Schmidt Mills': [],
+					'Schmidt Marrero': []}
+
+
+# Gets relevant values and calculates Schmidt number
+def get_schmidt(p, t, rh):
+	mills = props.get_d_12(p, t, 'Mills')
+	marrero = props.get_d_12(p, t, 'Marrero')
+
+	# Sometimes errors arise with coolprop from high relative humidities
+	try:
+		rho = 1/hap.HAPropsSI('Vha', 'P', p, 'T', t, 'RH', rh)
+	except ValueError:
+		return False, False
+
+	mu = props.get_mu(p, t, rh)
```

Comment:
This will need to be updated when you change the input of `get_mu` to take `t_dp` rather than `rh`. Just a heads up.